__std__/cells/plutus/packages/combined-plutus-haddock.nix

```diff
@@ -28,10 +26,9 @@ cell.library.combine-haddock {
         * "PlutusTx.Prelude": Haskell prelude replacement compatible with PLC.
         * "Plutus.Contract": Writing Plutus apps (off-chain code).
         * "Ledger.Constraints": Constructing and validating Plutus
-           transactions. Built on "PlutusTx" and
-           "Plutus.Contract".
+          transactions. Built on "PlutusTx" and "Plutus.Contract".
```

Comment:
Made these changes so that we could get rid of `# editorconfig-checker-disable-file`